stable/sonatype-nexus/templates/deployment.yaml

```diff
@@ -1,62 +1,149 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1beta2
 kind: Deployment
 metadata:
-  name: {{ template "fullname" . }}
+  name: {{ template "nexus.fullname" . }}
   labels:
-    app: {{ template "name" . }}
-    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
+{{ include "nexus.labels" . | indent 4 }}
 spec:
+  replicas: {{ .Values.replicaCount }}
   selector:
     matchLabels:
-      app: {{ template "name" . }}
+      app: {{ template "nexus.name" . }}
       release: {{ .Release.Name }}
-  replicas: {{ .Values.replicaCount }}
   template:
     metadata:
       labels:
-        app: {{ template "name" . }}
-        chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+        app: {{ template "nexus.name" . }}
         release: {{ .Release.Name }}
-        heritage: {{ .Release.Service }}
     spec:
-     {{- if .Values.nodeSelector }}
+      {{- if .Values.nexus.nodeSelector }}
       nodeSelector:
-{{ toYaml .Values.nodeSelector | indent 8 }}
-    {{- end }}
+{{ toYaml .Values.nexus.nodeSelector | indent 8 }}
+      {{- end }}
       containers:
-        - name: {{ .Chart.Name }}
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
-          imagePullPolicy: {{ .Values.image.pullPolicy }}
-          ports:
-            - containerPort: {{ .Values.service.internalPort }}
-            {{- if .Values.docker.enabled }}
-            - containerPort: {{ .Values.docker.port }}
-            {{- end }}
-          livenessProbe:
```

Comment:
Is there a particular reason the liveness/readiness probes were removed? If not, can you please add them back in?